department-of-veterans-affairs/caseflow
# Fixed timezone conversion bug for hearing time

Resolves #14779 
Resolves #14834

### Description
Fixes the incorrect conversion of the timezone on the user interface

### Acceptance Criteria

### Testing Plan
1. Shadow user `BVASYELLOW`
1. Navigate to the hearing schedule and select a Central hearing docket: http://localhost:3000/hearings/schedule
1. First make sure there is a hearing time set for the hearing
1. Click the 'Edit heearing details' link
1. Change the type of hearing to Virtual and ensure that the time is displayed correctly in EST
1. Navigate back to the daily docket
1. Change the hearing time and ensure that it displays the correct timezones for both the veteran and appellant
